bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -636,6 +645,151 @@ impl Room {
             Ok(())
         })
     }
+
+    pub fn send_image(
+        &self,
+        url: String,
+        thumbnail_url: String,
+        image_info: ImageInfo,
+    ) -> Result<(), RoomError> {
+        let mime_type = match &image_info.mimetype {
+            Some(mimetype) => {
+                mimetype.parse::<Mime>().map_err(|_| RoomError::InvalidAttachmentMimeType)
+            }
+            None => Err(RoomError::InvalidAttachmentMimeType),
+        }?;
+
+        let base_image_info =
+            BaseImageInfo::try_from(&image_info).map_err(|_| RoomError::InvalidAttachmentData)?;
+
+        let attachment_info = AttachmentInfo::Image(base_image_info);
+
+        let mut attachment_config;
+
+        if let Some(thumbnail_image_info) = image_info.thumbnail_info {
+            let thumbnail = self.build_thumbnail_info(thumbnail_url, thumbnail_image_info)?;
+            attachment_config = AttachmentConfig::with_thumbnail(thumbnail).info(attachment_info);
+        } else {
+            attachment_config = AttachmentConfig::new();
+            attachment_config = attachment_config.info(attachment_info);
+        }
```

Comment:
```suggestion
        let attachment_config = match image_info.thumbnail_info {
            Some(thumbnail_image_info) => {
                let thumbnail = self.build_thumbnail_info(thumbnail_url, thumbnail_image_info)?;
                AttachmentConfig::with_thumbnail(thumbnail).info(attachment_info);
            }
            None => {
                 AttachmentConfig::new().info(attachment_info)
            }
        };
```

Reply:
Nice, thanks 👍 Don't know why I didn't realise I could call info directly on that new().